docker/pypi/wmagent/wmagent-docker-run.sh

```diff
@@ -44,8 +44,8 @@ while getopts ":t:hp" opt; do
 done
 
 
-wmaUser=cmst1
-wmaOpts=" --user $wmaUser"
+wmaUser=$(id -un)
+userOpts=" --user $(id -u):$(id -g)"
```

Comment:
Well technically speaking, those are again `docker options` so the proper way is for them to be concatenated to the `dockerOpt` string variable.  The existence of the two variables:  `wmaOpts` and `dockerOpts` has different origin - it is their application target, rather their functional separation: 

* `dockerOpts` is the set of parameters passed to the docker engine 
* `wmaOpts` is the set of parameters passed to the service running inside the image (i.e. the entry point).

(The same applies to the MariaDB and CouchDB containers)